lib/cli-engine.js

```diff
@@ -21,10 +21,11 @@ var fs = require("fs"),
 
     assign = require("object-assign"),
     debug = require("debug"),
+    glob = require("glob"),
+    shell = require("shelljs"),
```

Comment:
We need to move this dependency into `dependencies` from `devDependencies` as it would fail for consumers.